### PR TITLE
docs: added 'sp1-ics07-tendermint' to use cases

### PR DESCRIPTION
**Pull request by srdtrk**
## Description

This proof of concept has been added to the cosmos org, and will continue to be worked on to serve as a tendermint light client in Solidity.

______

### PR author checklist:
- [ ] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests.
- [ ] Linked to GitHub issue.
- [ ] Updated code comments and documentation (e.g., `docs/`).
- [ ] Tagged *one* reviewer who will be the one responsible for shepherding this PR.

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ use cases.
 |  9 | Near Protocol                   |    Octopus    |    :100:     |                   [`near-ibc`][near-ibc]                    | :hourglass_flowing_sand: |
 | 10 | Tendermint Light Client         | Electron Labs |   `ics07`    | [`Electron-Labs/tendermint-rs`][electronlabs-tendermint-rs] | :hourglass_flowing_sand: |
 | 11 | Starknet                        |   Informal    |    :eyes:    |                           :eyes:                            |          :eyes:          |
+| 12 | SP1 Tendermint Light Client     |  Interchain   |   `ics07`    |        [`cosmos/sp1-ics07-tendermint`][sp1-tendermint]      | :hourglass_flowing_sand: |
```
Comment (Farhad-Shabani) [inline README.md:89]: Since I see the project is a PoC, should we use :hammer_and_pick: here? Is that employed in any devnet/testnet project?

**Review by Farhad-Shabani — COMMENTED** [review]
Cool! 🚀 
Thanks for adding! Just a question:

File: README.md (hunk 1 above)
Comment (srdtrk) [reply README.md:89]: I did push it on some testnets, but it is still WIP. Actually, we did decide to commit to it for now, and have partners launch with it if we can have the rest of IBC in Solidity.

**Review by Farhad-Shabani — APPROVED** [review]
:pray:

**Pull request merged**